files/galaxy/tpv/destinations.yml.j2

```diff
@@ -233,6 +233,32 @@ destinations:
       require:
         - cz-pulsar
 
+  pulsar_cz02_tpv:
+    inherits: pulsar_default
+    runner: pulsar_eu_cz02
+    max_accepted_cores: 32
+    max_accepted_mem: 128
+    min_accepted_gpus: 1
+    max_accepted_gpus: 1
+    env:
+      LC_ALL: C
+      SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      SINGULARITY_TMPDIR: "/storage/praha5-elixir/home/galaxyeu/singularity/tmp"
+      TMPDIR: "$SCRATCHDIR"
+      TMP: "$SCRATCHDIR"
+      TEMP: "$SCRATCHDIR"
+      ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.3.1"
```

Comment:
Just a tip: the newest version of AlphaFold expects one path that should contain the different AlphaFold database versions in separate subdirectories. Have a look at what [we had to do to get all versions to find the databases](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/468b63fcb70bf70637f244991706a99034dbb6f1/files/galaxy/tpv/tools.yml#L158-L166):

```yaml
      - if: helpers.tool_version_lt(tool, '2.3')
        params:
          singularity_run_extra_arguments: "--env ALPHAFOLD_DB=/data/db/databases/alphafold_databases/2.2/,ALPHAFOLD_USE_GPU=False"
      - if: helpers.tool_version_gte(tool, '2.3') and helpers.tool_version_lt(tool, '2.3.1+galaxy2')
        params:
          singularity_run_extra_arguments: "--env ALPHAFOLD_DB=/data/db/databases/alphafold_databases/2.3/,ALPHAFOLD_USE_GPU=False"
      - if: helpers.tool_version_gte(tool, '2.3.1+galaxy2')
        params:
          singularity_run_extra_arguments: "--env ALPHAFOLD_DB=/data/db/databases/alphafold_databases,ALPHAFOLD_USE_GPU=False"
```

Reply:
I had no idea. Thanks for letting me know. So, if I understood correctly, I shall aim for older versions (like 2.1.2+galaxy0), correct? I'm not sure we have a compatible DB version, but we can add it if needed (that will be handled after a test)

Reply:
Ideally we would have a way to let you have `gpus: 1` without interfering with our internal policy. I think @mira-miracoli is looking into that right now.